astropy/stats/funcs.py

```diff
@@ -1247,3 +1252,344 @@ def _kraft_burrows_nousek(N, B, CL):
         return _mpmath_kraft_burrows_nousek(N, B, CL)
 
     raise ImportError('Either scipy or mpmath are required.')
+
+
+def kuiper_FPP(D, N):
```

Comment:
It is generally frown upon to use uppercase for variable name (uppercase usually used for constant, not to be confused with `astropy.constants`). Is there a good reason to keep uppercase here?